phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -28,9 +30,12 @@
     @Value("${application.name}")
     private String applicationName;
 
+    @Autowired
```

Comment:
Theoritically yes, except that you have created a cyclic dependency which is forbidenn. 
The right way to solve this problem in to create an extra module with a new configuration file